static/app/utils/metrics.tsx

```diff
@@ -97,28 +106,43 @@ export function useMetricsData({
   const useCase = getUseCaseFromMri(mri);
   const field = op ? `${op}(${mri})` : mri;
 
-  const {start, end} = getUTCTimeRange(timeRange);
+  const {start, end, period} = useMemo(
+    () => getUTCTimeRange(datetime.start, datetime.end, datetime.period),
+    [datetime.period, datetime.start, datetime.end]
+  );
```

Comment:
Do we still need this now that we use page filters? I don't see this anywhere in the app so it should support timezones ootb.